applications/dashboard/controllers/class.settingscontroller.php

```diff
@@ -568,9 +568,8 @@ public function branding() {
                 'LabelCode' => t('Defer Javascript Loading'),
                 'Control' => 'toggle',
                 'Default' => true,
-                'Description' => t('This setting loads the page before executing Javascript which can improve your SEO.<br>
-                    <strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong> ' .
-                    anchor(t('More information'), 'https://success.vanillaforums.com/kb/articles/140-defer-javascript-loading-feature')),
+                'Description' => t('This setting loads the page before executing Javascript') .
```

Comment:
Not a sentence when the translation is a sentence. You also need to put some whitespace between this and the concat.

Reply:
How much whitespace? There's currently 1 space between the two because the translation string has a trailing space (I don't know if that's kosher or not, but that's how it was originally).

<img width="580" alt="Screen Shot 2020-03-18 at 9 45 08 AM" src="https://user-images.githubusercontent.com/17052116/76967183-73616a00-68fd-11ea-8c03-89fd56983f52.png">

Reply:
Yeah no trailing space in the translation. You can put a space outside of the translation though.